build-docker-image/action.yaml

```diff
@@ -110,8 +110,15 @@ runs:
         fi
 
 
+        # for scheduled runs we do not have the default_branch in the event
+        if [ "${{ github.event.repository.default_branch }}"  == "" ] || [ "${{ github.event.repository.default_branch }}"  == "${GITHUB_REF#refs/*/}" ];then
+          DEFAULT_BRANCH=
```

Comment:
is it really just "DEFAULT_BRANCH=" and then nothing? No empty string or anything?